### ISSUE TITLE
[Feature Request]  DatePicker component not automatically switches back to the date view after selecting a month and year.

### ISSUE THREAD

**Issue by vanphat304** [nextui-org/nextui#4002]
### Is your feature request related to a problem? Please describe.

NextUI's DatePicker component not automatically switches back to the date view after selecting a month and year.

### Describe the solution you'd like

NextUI's DatePicker component that automatically switches back to the date view after selecting a month and year.

### Describe alternatives you've considered

NextUI's DatePicker component that automatically switches back to the date view after selecting a month and year.

**Comment by ds-spec** [comment]
Hi! I'd like to clarify the issue to make sure I understand it correctly before working on it.

It seems that when using the DatePicker component with both month and year pickers, after selecting a specific month and year, the component doesn’t automatically switch back to the date view. Instead, users have to press on the button to navigate back manually. This adds an extra step and interrupts the flow.

Is the goal of this issue to modify the DatePicker so that it automatically returns to the date view right after a month and year are selected?

Thank you for the clarification!

**Comment by vanphat304** [comment]
yes , i want automatically returns to the date view right after a month and year are selected . thank you for supports
